### DISCUSSION TITLE
MKR 1310 Setup

### DISCUSSION THREAD

**Discussion by engeen-nl** [sandeepmistry/arduino-LoRa#501] [Q&A] [answered]
I'm attempting to use the SX1276 on the MKR1310 for FSK modulation. When I try the SX127x_FSK_Modem example I'm getting a
`[SX1276] Initiating ... failed, code -2`
-2 means ERR_CHIP_NOT_FOUND
I'm sure using the right modem (SX1276 address 0x12)
Selected pins:
`SX1276 radio = new Module(LORA_IRQ_DUMB, LORA_IRA, LORA_RESET, RADIOLIB__NC, SPI1);`
I also tried LORA_BOOT0 but no succes. I also tried the reset mechanism as used by the LoRa library of sandeepmistry, but this also doesn't work. Anyone any ideas? Help would be greatly appreciated.

**Comment by IoTThinks** [comment, marked as answer]
This library doesn't support FSK.
It seems that you are using RadioLib, so you should go there to ask instead?

**Reply by engeen-nl** [reply]
Apologies, it was late last night. Indeed this was meant to go to Radiolib. arduino-LoRa works like a charm in this regard! Just because of a customer scenario I need to try FSK to see if it will work better. If you have any ideas after all, please let me know at /jgromes/RadioLib/discussions/343
